.github/workflows/build.yml

```diff
@@ -34,6 +34,12 @@ defaults:
 env:
   BUILD_ID: ${{ inputs.buildId }}
   BASHBREW_ARCH: ${{ inputs.bashbrewArch }}
+  BASHBREW_IMAGE_SIGNER: 'docker/image-signer-verifier:0.5.15@sha256:322351455cff9ce6ea3bc61e531b1ce26c7c93ff5283b64a8c65bbd1122ed28a'
+  BASHBREW_REFERRERS_REPO: oisupport/referrers  # repo to store attestation artifacts
+  BASHBREW_TUF_META: docker/tuf-metadata:latest  # TUF trust root for verifying DOI policy required for generating Verification Summary Attestations (VSA)
+  BASHBREW_TUF_TARGETS: docker/tuf-targets  # see https://github.com/docker/tuf and https://theupdateframework.io/
+  BASHBREW_SIGNING_POLICY: docker-official-images-full # policy id for rego stored in TUF to verify DOI attestations for VSA generation (https://github.com/docker/tuf/blob/main/targets/doi/policy-full.rego)
+
 jobs:
   build:
```

Comment:
We'll need to pass our [builder-id to BuildKit](https://github.com/moby/buildkit/blob/master/docs/attestations/slsa-provenance.md#parameters) to meet SLSA build level 2 in this PR. For GHA, it is pretty standard to use https://github.com/slsa-framework/github-actions-buildtypes/tree/main/workflow/v1#builder